src/commands/CommandSample.disable

```diff
@@ -0,0 +1,30 @@
+import { Command } from "../Command";
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder } from "@discordjs/builders";
+
+// set the class name, the export at the bottom and the file name to your desired command name
+class CommandName extends Command {
+    constructor() {
+        super("bunny"); // the name under which the bot internally stores your command
+    }
+
+    async execute(interaction: CommandInteraction): Promise<void> {
+        // put the logic of your command here
+        // for example:
+        const num = interaction.options.getInteger("amount") || 1;
+        await interaction.reply("🐇".repeat(num));
+    }
+
+    register() {
+        // set the name and description shown in discord here (these are necessary)
+        // (name: string.length = 1-32, description string.length = 1-100)(both lowercase!)
```

Comment:
Description doesn't need to be lowercase?

Reply:
yup checks out
thanks

Reply:
```suggestion
        // (name: string.length = 1-32, description string.length = 1-100)(the name has to be lowercase!)
```